test/helpers.spec.ts

```diff
@@ -854,5 +862,33 @@ Try adjusting maxTime or maxRetries parameters for faker.helpers.unique().`)
         expect(store).toEqual({ 'with conflict: 0': 'with conflict: 0' });
       });
     });
+
+    describe('multiple()', () => {
+      it('should generate values from the function', () => {
```

Comment:
```suggestion
      it('should generate values from the function with a default length of 3', () => {
```

Just to make sure that the test title specifies exactly what it does.